code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -119,9 +119,6 @@
 	var/scp_013_instance = FALSE
 	var/scp_013_stage = 0
 
-	// SCP-173
-	var/scp173_killed = FALSE
-
```

Comment:
Why was the scp173_killed variable even a thing?
It literally never did anything